CenterForOpenScience/osf.io
# [Feature] Fix reg withdraw hook for django [OSF-7545]

## AFTER THIS GOES IN THE MIGRATION SCRIPT FOR REGISTRIES SHOULD PROBABLY BE RE-RUN TO ENSURE SHARE HAS ALL THE RIGHT DATA. Script: osf.io/scripts/migration/migrate_share_registration_data.py




## Purpose
Withdrawing does not manually call on node updated actions on the registration (only on the node), causing withdrawals to not hit the share hook properly.



## Changes
Add a manual call to the hook in the retraction model at the end of on complete


## Side effects
None known



## Ticket
[#OSF-7545]
https://openscience.atlassian.net/browse/OSF-7545



## QA Notes

Make sure retracting a registration populates the withdraw tag on Registries and that it generally doesn't break anything :)

from https://github.com/CenterForOpenScience/osf.io/pull/6944
